web/core/components/issues/issue-detail-widgets/widget-collapsible-button.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+import React, { FC } from "react";
+import { DropdownIcon } from "@plane/ui";
+// helpers
+import { cn } from "@/helpers/common.helper";
+
+type Props = {
+  isOpen: boolean;
+  title: string;
+  hideChevron?: boolean;
+  indicatorElement?: React.ReactNode;
+  actionItemElement?: React.ReactNode;
+};
+
+export const WidgetCollapsibleButton: FC<Props> = (props) => {
+  const { isOpen, title, hideChevron = false, indicatorElement, actionItemElement } = props;
+  return (
+    <div className="flex items-center justify-between gap-3 h-12 px-2.5 py-3 border-b border-custom-border-100">
```

Comment:
We should create a button component for this.

Reply:
Done, 
I've created a CollapsibleButton component in plane ui package.

Reply:
Done